arctic/store/bson_store.py

```diff
@@ -30,7 +30,7 @@ def __init__(self, arctic_lib):
     def initialize_library(cls, arctic_lib, hashed=True, **kwargs):
         logger.info("Trying to enable sharding...")
         try:
-            enable_sharding(arctic_lib.arctic, arctic_lib.get_name(), hashed=hashed)
+            enable_sharding(arctic_lib.arctic, arctic_lib.get_name(), hashed=hashed, key='_id')
```

Comment:
If you're sharding by `_id` I think you always want `hashed=True` (as `ObjectId`s are ordered, so range based sharding will cause them to move about, and all writes will happen to a single shard).